tests/test_automatic_payout_transactions.py

```diff
@@ -0,0 +1,89 @@
+from utils import stripe_obj_to_dict, client, midnight
+from tap_tester import runner, connections
+from base import BaseTapTest
+
+def get_payouts():
+    # list of all data to return
+    data = []
+    # api call of 1st page
+    stripe_obj = client["payouts"].list(limit=100, created={"gte": midnight})
+    dict_obj = stripe_obj_to_dict(stripe_obj)
+    # add data
+    data += dict_obj['data']
+
+    # loop over rest of the pages and collect data
+    while dict_obj.get("has_more"):
+        stripe_obj = client["payouts"].list(limit=100, created={"gte": midnight}, starting_after=dict_obj.get('data')[-1].get('id'))
+        dict_obj = stripe_obj_to_dict(stripe_obj)
+        data += dict_obj['data']
+
+    # send data
+    return data
+
+class AutomaticPayoutTransactionTest(BaseTapTest):
+    """
+        Test case to verify that we only collect payout_transactions for payouts containing "automatic" field as "True"
+    """
+
+    @staticmethod
+    def name():
+        return "tt_stripe_automatic_payout_transactions"
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Add functional comment to describe the reason for adding this function.

Reply:
Added comment.